honeybee_schema/radiance/asset.py

```diff
@@ -0,0 +1,156 @@
+"""SensorGrid and Sensor Schema"""
+from pydantic import Field, constr
+from typing import List
+from enum import Enum
+from .._base import NoExtraBaseModel
+from ._base import IDdRadianceBaseModel
+
+
+class Sensor(NoExtraBaseModel):
+    """A single Radiance of sensors."""
+
+    type: constr(regex='^Sensor$') = 'Sensor'
+
+    pos: List[float] = Field(
+        ...,
+        description="Position of sensor in space as an array of (x, y, z) values.",
+        min_items=3,
+        max_items=3
+    )
+
+    dir: List[float] = Field(
+        ...,
+        description="Direction of sensor as an array of (x, y, z) values.",
+        min_items=3,
+        max_items=3
+    )
+
+
+class SensorGrid(IDdRadianceBaseModel):
+    """A grid of sensors."""
+
+    type: constr(regex='^SensorGrid$') = 'SensorGrid'
+
+    sensors: List[Sensor] = Field(
+        ...,
+        description="A list of sensors that belong to the grid."
+    )
+
+    room_identifier: str = Field(
+        None,
+        regex=r'[A-Za-z0-9_-]',
+        min_length=1,
+        max_length=100,
+        description='Optional text string for the Room identifier to which this '
+            'SensorGrid belongs. This will be used to narrow down the number of '
+            'aperture groups that have to be run with this sensor grid. If None, '
+            'the grid will be run with all aperture groups in the model.'
```

Comment:
This should be aligned with description. I have made the same mistake but this is an input value not a docstring.
There are multiple cases like this but I only leave the comment for this one.

```suggestion
        'SensorGrid belongs. This will be used to narrow down the number of '
        'aperture groups that have to be run with this sensor grid. If None, '
        'the grid will be run with all aperture groups in the model.'
```

Reply:
Thanks for catching that. I will fix it.